portable-atomic-util/src/lib.rs

```diff
@@ -60,6 +60,14 @@ See [#1] for other primitives being considered for addition to this crate.
 #![allow(clippy::inline_always)]
 // docs.rs only (cfg is enabled by docs.rs, not build script)
 #![cfg_attr(docsrs, feature(doc_cfg))]
+// Enable custom unsized coercions (custom feature) if enabled & possible with current Rust compiler version (see build.rs)
+#![cfg_attr(
+    all(portable_atomic_coerce_unsized, not(portable_atomic_no_unstable)),
+    feature(coerce_unsized, unsize)
+)]
+
+#[cfg(all(portable_atomic_no_unstable, portable_atomic_coerce_unsized))]
+compile_error!("`portable_atomic_coerce_unsized` cfg requires Rust nightly");
```

Comment:
I don't see why portable_atomic_no_unstable cfg is needed, since enabling the feature in stable is an error automatically.

Reply:
I would personally prefer that there is a more human-readable error message in case someone tries using this new `--cfg` feature with non-nightly toolchain. Any idea for a better or easier way to do this?

Reply:
If the cfg name contains "unstable", I think it is implied.

Reply:
portable_atomic_no_unstable cfg is now removed